JustSaying/IAmJustSaying.cs

```diff
@@ -8,7 +8,7 @@
 
 namespace JustSaying
 {
-    public interface IAmJustSaying : IMessagePublisher
+    public interface IAmJustSaying : IMessagePublisher, IDisposable

```

Comment:
See comment, I think we shouldn't own a `CancellationTokenSource`, so we wouldn't need to implement `IDisposable`